tasks.go

```diff
@@ -125,24 +125,31 @@ func buildHooks(yamlData TsuruYaml, envs []bind.EnvVar) error {
 	return execScript(cmds, envs, os.Stdout)
 }
 
-func readProcfile() (string, error) {
-	procfilePath := fmt.Sprintf("%s/%s", defaultWorkingDir, "Procfile")
-	f, err := filesystem().Open(procfilePath)
+func readProcfile(path string) (string, error) {
+	CR := byte(13)
+	f, err := filesystem().Open(fmt.Sprintf("%v/Procfile", path))
 	if err != nil {
 		return "", err
 	}
 	defer f.Close()
 	procfile, err := ioutil.ReadAll(f)
+	procfile = append(procfile, CR)
```

Comment:
Wouldn't it be easier here to simply call `procfile = bytes.Replace(procfile, []byte("\r\n"), []byte("\n"), -1)` instead of manually iterating on each byte? Do you see any problems with this approach?

Reply:
No problem, rs

Reply:
:up: